benchrun.sh

```diff
@@ -9,7 +9,7 @@ else
     BENCH_DIR="./bench-results"
 fi
 
-racket -y tests/bench-paper.rkt | tee bench-log-ex.txt
+xvfb-run racket -y tests/bench-paper.rkt | tee bench-log-ex.txt
```

Comment:
is that a problem for those not running under docker?

Reply:
I believe it will be a problem for those not running under linux or docker. IIUC, the `xvfb-run` is a bash script available in linux but incompatible w/MacOS. It might be that Raffi has already side-stepped this issue with https://github.com/namin/staged-miniKanren/pull/26 , and that that will solve the problem. But not sure.